crates/bevy_ui/src/flex/convert.rs

```diff
@@ -1,67 +1,96 @@
 use crate::{
-    AlignContent, AlignItems, AlignSelf, Display, FlexDirection, FlexWrap, JustifyContent,
-    PositionType, Size, Style, UiRect, Val,
+    AlignContent, AlignItems, AlignSelf, Breadth, Display, FlexDirection, FlexWrap, JustifyContent,
+    JustifySelf, PositionType, Style, Val,
 };
+use taffy::prelude::{LengthPercentage, LengthPercentageAuto, Rect};
+use taffy::style::Dimension;
 
-pub fn from_rect(
-    scale_factor: f64,
-    rect: UiRect,
-) -> taffy::geometry::Rect<taffy::style::Dimension> {
-    taffy::geometry::Rect {
-        left: from_val(scale_factor, rect.left),
-        right: from_val(scale_factor, rect.right),
-        top: from_val(scale_factor, rect.top),
-        bottom: from_val(scale_factor, rect.bottom),
+fn from_breadth(scale_factor: f64, breadth: Breadth) -> LengthPercentage {
+    match breadth {
+        Breadth::Percent(value) => LengthPercentage::Percent(value / 100.0),
```

Comment:
Feels oddly inconsistent that the one type is 0..1 and the other is 0..100

Reply:
Ditto for Val and Dimension below.

Reply:
I prefer 0..1, but it would need to be renamed I think and I can't come up with anything as intuitive as `Percent`

Reply:
Yeah I'm fine with either choice so long as we're consistent across our types.

Reply:
Yeah, I was thinking something like `PercentAsFloat` or `PercentAsDecimal`. `Val::Percent(1.0)` is definitely confusing! Taffy currently has the same issue.